site/docs/.vitepress/config.ts

```diff
@@ -43,6 +44,8 @@ export default defineConfig({
         aliases: ["asciiart", "ascii", "ftl", "log", "procfile", "text"],
         ...plaintext,
       },
+      // @ts-ignore
```

Comment:
Why does this not compile?

Reply:
```ansi
Type '{ name: string; scopeName: string; fileTypes: string[]; uuid: string; patterns: ({ comment: string; match: string; name: string; captures: { "1": { name: string; }; }; begin?: undefined; beginCaptures?: undefined; patterns?: undefined; end?: undefined; endCaptures?: undefined; } | { ...; } | { ...; } | { ...; })[]; ...' is not assignable to type 'LanguageInput'.
  Type '{ name: string; scopeName: string; fileTypes: string[]; uuid: string; patterns: ({ comment: string; match: string; name: string; captures: { "1": { name: string; }; }; begin?: undefined; beginCaptures?: undefined; patterns?: undefined; end?: undefined; endCaptures?: undefined; } | { ...; } | { ...; } | { ...; })[]; ...' is not assignable to type 'LanguageRegistration'.
    Types of property 'repository' are incompatible.
      Type '{ interpolation: { comment: string; begin: string; beginCaptures: { "1": { name: string; }; }; patterns: { match: string; captures: { "1": { name: string; }; "2": { name: string; }; }; }[]; end: string; endCaptures: { "1": { name: string; }; }; }; variable: { ...; }; "escape-characters": { ...; }; }' is not assignable to type 'IRawRepository'.
        Type '{ interpolation: { comment: string; begin: string; beginCaptures: { "1": { name: string; }; }; patterns: { match: string; captures: { "1": { name: string; }; "2": { name: string; }; }; }[]; end: string; endCaptures: { "1": { name: string; }; }; }; variable: { ...; }; "escape-characters": { ...; }; }' is missing the following properties from type 'IRawRepositoryMap': $self, $base
```

Reply:
Sure. Give me a couple of days

Reply:
Their grammar list doesn't seem to strictly follow the type either (see https://github.com/shikijs/textmate-grammars-themes/). To make sure it meets the required properties, I added an empty value to it at https://github.com/grammyjs/website/pull/1083/commits/fbd3565406b0315857c6b865b32f9a58f3c0246e.